examples/player/simid_player.html

```diff
@@ -42,6 +42,27 @@
       simidPlayer.playAd();
     }
 
+    function closeAd() {
+      if (!simidPlayer) {
+        initAd();
+      }
+      simidPlayer.closeAd();
+    }
+
+    function skipAd() {
+      if (!simidPlayer) {
+        initAd();
```

Comment:
I don't get it.  So if the user clicks skip ad, he could end up initializing an ad and then immediately skipping it?

Seems like what you want is to just have the skip ad button not even available unless and ad has been initialized.  Then you could probably just get rid of this if statement.

Reply:
fixed